src/main/java/com/czertainly/api/model/connector/notification/NotificationRecipientDto.java

```diff
@@ -0,0 +1,27 @@
+
+package com.czertainly.api.model.connector.notification;
+
+import com.czertainly.api.model.client.attribute.RequestAttributeDto;
+import com.czertainly.api.model.client.attribute.ResponseAttributeDto;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class NotificationRecipientDto {
+    @Schema(description = "Recipient type", requiredMode = Schema.RequiredMode.REQUIRED)
+    private NotificationRecipientType type;
+
+    @Schema(description = "Recipient name", requiredMode = Schema.RequiredMode.REQUIRED)
+    private String name;
+
+    @Schema(description = "Recipient email", requiredMode = Schema.RequiredMode.REQUIRED)
```

Comment:
I think for group this is not available right now.

Reply:
It is available, but needs to be added for role. Will be resolved in other PR.